intake/cli/server/__main__.py

```diff
@@ -27,6 +27,8 @@ def main(argv=None):
     parser = argparse.ArgumentParser(description='Intake Catalog Server')
     parser.add_argument('-p', '--port', type=int, default=conf['port'],
                         help='port number for server to listen on')
+    parser.add_argument('--list-entries', action='store_true',
```

Comment:
Good idea!

Also, flatten yes/no may be a good one.

Reply:
Thanks. I haven't quite wrapped my brain around how that would go so I am punting this to [a new issue](https://github.com/ContinuumIO/intake/issues/202). Feel free to consider it "assigned" to me.